google/cloud/bigquery/_helpers.py

```diff
@@ -419,9 +419,23 @@ def _record_field_to_json(fields, row_value):
     Returns:
         Mapping[str, Any]: A JSON-serializable dictionary.
     """
-    record = {}
     isdict = isinstance(row_value, dict)
```

Comment:
If it's already a dictionary, is there any need for additional processing?

Reply:
We do, because we need to explicitly exclude any `None` values from the subsequent request ([commit](https://github.com/googleapis/python-bigquery/commit/60926a2e954b4bc2f5796b5ade339342c574b481)).